templates/OperationResponseBodySnippet.stencil

```diff
@@ -19,7 +18,5 @@ case let .success(data):
     }
     {% include "OperationResponseExceptionSnippet.stencil" %}
 case let .failure(error):
-    DispatchQueue.main.async {
-        completionHandler(.failure(error), httpResponse)
-    }
+    {% include "OperationResponseFailureSnippet.stencil" %}
```

Comment:
What's the difference between:
- OperationResponseFailureSnippet
- OperationResponseExceptionSnippet

The sound very very smiliar.

Reply:
ResponseException is for the 'exception' array in the code mode.
ResponsFailure is when the result from the http handler returns failure.